src/main/app/controller/AppRequest.ts

```diff
@@ -24,6 +24,7 @@ export interface AppSession extends Session {
   lang: string | undefined;
   errors: FormError[] | undefined;
   addresses: [];
+  fileErrors: any[];
```

Comment:
Unchanged from DTSSTCI-535

Reply:
How is this related to https://tools.hmcts.net/jira/browse/DTSSTCI-535?

Reply:
Sorry my mistake 353.